WordPress.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -14,9 +14,9 @@
         "package": "XCUITestHelpers",
         "repositoryURL": "https://github.com/Automattic/XCUITestHelpers",
         "state": {
-          "branch": null,
-          "revision": "47d688b0b2a6356361a16836c4d23f12d6643a73",
-          "version": "0.1.0"
+          "branch": "add-helpers-from-woo",
```

Comment:
I identified only a couple of utilities that were used only in this app, and I didn't extract them. The extraction work already done for WooCommerce allowed the removal of a good chunk of duplicated code here anyway.